docs/overrides/main.html

```diff
@@ -1,5 +1,5 @@
 {% extends "base.html" %}
 
 {% block announce %}
-   SQLDelight 2.0 is now available! See <a href="upgrading-2.0">the docs</a> on migrating.
+   SQLDelight 2.0 is now available! See <a href="/sqldelight/upgrading-2.0">the docs</a> on migrating.
```

Comment:
Annoyingly, this won't work because our docs are versioned.

e.g. https://cashapp.github.io/sqldelight/**2.0.0-rc02**/upgrading-2.0

Let me check if there's a way to use the version macro in here.

Reply:
Nevermind, couldn't get it to work.

I added a redirect in our gh-pages setup instead.